examples/github/github-eventlistener-interceptor.yaml

```
@@ -17,6 +17,53 @@ spec:
         - cel:
             filter: "body.action in ['opened', 'synchronize', 'reopened']"
       bindings:
-        - ref: github-binding
+        - ref: github-pr-binding
       template:
         name: github-template
+---
+apiVersion: triggers.tekton.dev/v1alpha1
+kind: TriggerBinding
+metadata:
+  name: github-pr-binding
+spec:
+  params:
+    - name: gitrevision
+      value: $(body.pull_request.head.sha)
+    - name: gitrepositoryurl
+      value: $(body.repository.clone_url)
+
+---
+apiVersion: triggers.tekton.dev/v1alpha1
+kind: TriggerTemplate
+metadata:
+  name: github-template
+spec:
+  params:
+    - name: gitrevision
+    - name: gitrepositoryurl
+  resourcetemplates:
+    - apiVersion: tekton.dev/v1alpha1
+      kind: TaskRun
+      metadata:
+        generateName: github-run-
+      spec:
+        taskSpec:
+          inputs:
+            resources:
+              - name: source
+                type: git
+          steps:
+            - image: ubuntu
+              script: |
+                #! /bin/bash
+                ls -al $(inputs.resources.source.path)
+        inputs:
+          resources:
+            - name: source
+              resourceSpec:
+                type: git
+                params:
+                  - name: revision
+                    value: $(params.gitrevision)
```

Nice fix :+1: how its use to work earlier :thinking: 

small changes related to `tt.params`
```suggestion
                    value: $(tt.params.gitrevision)
```

---

Done!
We were using a mock/fake body to test this example out. Turns out that body was wrong  -- it was for the push event not pull_request too. So, the example worked. When tried with a real GitHub PR, it failed 😅